README.md

```diff
@@ -30,7 +30,7 @@ To build Neon, the following are required:
 - [SCons](http://scons.org)
 - A C++11 compiler (a modern `gcc`, `clang`, or Visual Studio 2013 or later)
 
-Current master branch build status: [![Build Status](https://travis-ci.org/ghewgill/neon-lang.svg?branch=master)](https://travis-ci.org/ghewgill/neon-lang)
+Current master branch build status: [![Build Status](https://travis-ci.org/ghewgill/neon-lang.svg?branch=master)](https://travis-ci.org/ghewgill/neon-lang) (https://ci.appveyor.com/api/projects/status/github/ghewgill/neon-lang?branch=master&svg=true)
```

Comment:
This isn't right. It needs to be a button, just like the travis one. You can see the result at [your appveyor branch](https://github.com/gitlarryf/neon-lang/tree/appveyor), which clearly doesn't show a button here.